bundles/automation/org.eclipse.smarthome.automation.module.core/ESH-INF/automation/moduletypes/ItemTriggers.json

```diff
@@ -128,6 +138,40 @@
                "required":true
             },
             {  
+               "name":"oldState",
```

Comment:
No it is not, it is the core.ItemStateChangeTrigger. It is an input parameter to allow reducing the trigger to only trigger if the previous state was "oldState".